cli/lib/png2src.js

```diff
@@ -26,10 +26,14 @@ const %idiomaticName%: [u8; %length%] = [ %bytes% ];
 `,  
 
     go: 
-`const %name%Width = %width%;
-const %name%Height = %height%;
-const %name%Flags = %flags%; // %flagsHumanReadable%
-var %name% = [%length%]byte { %bytes% };
+`// %name%
+const (
+    %name%Width = %width%
+    %name%Height = %height%
+    %name%Flags = %flags% // %flagsHumanReadable%
+)
+
+var %name% = [%length%]byte { %bytes% }
```

Comment:
Last nitpick! I think this should remain:

```suggestion
`const %name%Width = %width%
const %name%Height = %height%
const %name%Flags = %flags% // %flagsHumanReadable%
var %name% = [%length%]byte { %bytes% }
```

- More compact output
- Nicer looking output when using multiple images: `w4 png2src assets/*.png --go`. Images are separated by blank lines

Reply:
Actually I changed it to this *because* this would be more eye-friendly. But I guess that's my personal opinion.